tools/topology/topology2/include/pipelines/cavs/host-copier-gain-src-mixin-playback.conf

```diff
@@ -0,0 +1,300 @@
+#
+# FE playback pipeline: host-copier-gain-mixin-playback
+#
+# All attributes defined herein are namespaced
+# by alsatplg to "Object.Pipeline.host-copier-gain-src-mixin-playback.N.attribute_name"
+#
+# Usage: host-copier-gain-src-mixin-playback pipeline object can be instantiated as:
+#
+# Object.Pipeline.host-copier-gain-src-mixin-playback."N" {
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/common/audio_format.conf>
+<include/components/copier.conf>
+<include/components/mixin.conf>
+<include/components/pipeline.conf>
+<include/components/gain.conf>
+<include/components/src.conf>
+
+Class.Pipeline."host-copier-gain-src-mixin-playback" {
+
+	DefineAttribute."index" {}
+
+	<include/pipelines/pipeline-common.conf>
+
+	attributes {
+		!constructor [
+			"index"
+		]
+
+		!immutable [
+			"direction"
+		]
+
+		#
+		# host-copier-gain-src-mixin-playback objects instantiated within the same
+		# alsaconf node must have unique pipeline_id attribute
+		#
+		unique	"instance"
+	}
+
+	Object.Widget {
+		copier."1" {
+			copier_type	"host"
+			type	"aif_in"
+			node_type $HDA_HOST_OUTPUT_CLASS
+			num_audio_formats 8
+			num_input_audio_formats 8
+			num_output_audio_formats 8
+
+			Object.Base.audio_format.1 {
```

Comment:
@ranj063 @juimonen @aiChaoSONG Is there a way to make this more compact? These are not yet all standard rates 8 - 192 kHz, and I'd need to add also 16 and 24 bit. This list of formats repeats until SRC is reached in the pipeline.

Reply:
I don't have a way to pack it, just list all cases

Reply:
Yep, I will include your .conf files for it with all 42 formats.